_config.yml

```diff
@@ -1,5 +1,6 @@
 title: The Lounge &mdash; Self-hosted web IRC client
-description: "The official website for The Lounge, the self-hosted web IRC client."
+description: "The official website for The Lounge, the self-hosted web IRC client for the modern world."
+extra_description: "The Lounge is a web-based IRC client for the modern world!"
```

Comment:
Just to make sure I understand, this is because, if the page has a description, you want "page description + extra description", and if not, just "site description"?

If so, I think we could just go with: if page has description, go "page description + site description", otherwise just "site description". It's kinda odd to have both, and I don't mind the extra-wordiness of `description` 🤷‍♂️